modelerfour/prenamer/prenamer.ts

```diff
@@ -192,13 +192,11 @@ export class PreNamer {
 
       // handle operation parameters
       for (const parameter of collisions) {
-        console.error(`Collision : ${parameter.language.default.name}`);
```

Comment:
Seems like this would have been useful.  Was it spewing too much output?

Reply:
Just not necessary- it was some debug output. Collisions are commonplace on complex models.